.eslintrc

```diff
@@ -2,6 +2,9 @@
     "parserOptions": {
         "ecmaVersion": 2015
     },
+    "env": {
+        "es6": true
```

Comment:
We can also just add Set to globals, but I don't see why we wouldn't have it here, since we also allow ES6 syntax with parserOptions.